envs/conda_build_config.yaml

```diff
@@ -1,13 +1,13 @@
 abseil_cpp:
   - 20200225
 absl_py:
-  - 0.10.*
+  - 0.12.*
 arrow:
   - 3.0.0
 astunparse:
   - 1.6
 attrs:
-  - 19
+  - 19.1.*
```

Comment:
Why tighten this pin?

Reply:
Yes, we do not need this. I will revert this change.